### DISCUSSION TITLE
Handling "ownership" of resources that have been split

### DISCUSSION THREAD

**Discussion by BNAndras** [ohmg-dev/OldInsuranceMaps#128] [Ideas]
**Describe the problem**
In the recent activity feed, changes to a document or layer are attributed to the volume owner even if they were done by someone else.

**Expected behavior**
Changes to documents and layers should be attributed to the registered user who performed the operation (e. g. splitting an operation or georeferencing a layer).

**Comment by mradamcox** [comment]
This wasn't a bug, per se, but it definitely warrants revisiting. Who should be the "owner" of a document when it is prepared? What does the "owner" property mean, anyway? It would be best to address the latter question first. We can always present the name of who prepared a document without changing the "owner" attribute.

**Comment by mradamcox** [comment]
Lots of flexibility now that local Document/Layer models are in use, and may even be combined further into a single model. Should there be an "owner" at the Resource level at all? I'm thinking no, because the Volume level of the content hierarchy will likely be abstracted to something a bit more generic, Item, and that level is where "ownership" actually makes the most sense. There is probably a better term for this anyway...

**Comment by mradamcox** [comment]
Closing as the data model has changed enough that this topic is outdated.
